integration-cli/docker_cli_rmi_test.go

```diff
@@ -13,27 +14,21 @@ func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
 	errSubstr := "is using it"
 
 	// create a container
-	out, _, err := dockerCmdWithError("run", "-d", "busybox", "true")
-	if err != nil {
-		c.Fatalf("failed to create a container: %s, %v", out, err)
-	}
+	out, _ := dockerCmd(c, "run", "-d", "busybox", "true")
 
 	cleanedContainerID := strings.TrimSpace(out)
 
 	// try to delete the image
-	out, _, err = dockerCmdWithError("rmi", "busybox")
-	if err == nil {
-		c.Fatalf("Container %q is using image, should not be able to rmi: %q", cleanedContainerID, out)
-	}
-	if !strings.Contains(out, errSubstr) {
-		c.Fatalf("Container %q is using image, error message should contain %q: %v", cleanedContainerID, errSubstr, out)
-	}
+	out, _, err := dockerCmdWithError("rmi", "busybox")
+	// Container is using image, should not be able to rmi
+	c.Assert(err, checker.NotNil)
+	// Container is using image, error message should contain errSubstr
+	c.Assert(out, checker.Contains, errSubstr, check.Commentf("Container: %q", cleanedContainerID))
 
 	// make sure it didn't delete the busybox name
 	images, _ := dockerCmd(c, "images")
-	if !strings.Contains(images, "busybox") {
-		c.Fatalf("The name 'busybox' should not have been removed from images: %q", images)
-	}
+	// The name 'busybox' should not have been removed from images
+	c.Assert(images, checker.Contains, "busybox")
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?